applications/rag/frontend/outputs.tf

```diff
@@ -12,6 +12,6 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-output "domain" {
-  value = var.url_domain_addr != "" ? var.url_domain_addr : "${google_compute_global_address.default[0].address}.nip.io"
-}
+output "frontend_domain" {
+  value = var.add_auth ? module.iap_auth[0].frontend_domain : var.url_domain_addr
```

Comment:
`var.url_domain_addr` will not make sense here.. because this variable is only set for IAP purpose.

if IAP is false.. this var will be false always.. instead lets fetch ip from frontend-service (if its loadbalancer type..) or else just a text that "internal_access_only"

Reply:
Good point, let me make it as jupyter